common/fpga.make

```diff
@@ -228,14 +232,18 @@ u-boot-src: $(U_BOOT_SRC)
 .PHONY: u-boot-src
 # -----------------------------------------------------------------------------------
 
-$(DEVTREE_DTB): $(SDK_EXPORT) $(TARGET_DTS)
+$(DEVTREE_DTB): $(SDK_EXPORT) $(TARGET_DTS) | $(DEVTREE_DTC)
 	cp $(TARGET_DTS) $(DEVTREE_DTS)/
 	sed -i '/dts-v1/d' $(DEVTREE_DTS)/system-top.dts
 	gcc -I dts -E -nostdinc -undef -D__DTS__ -x assembler-with-cpp \
 	  -o $(DEVTREE_DTS)/system-top.dts.tmp $(DEVTREE_DTS)/system-top.dts
 	@echo "Building DEVICE TREE blob ..."
 	$(DEVTREE_DTC) -f -I dts -O dtb -o $@ $(DEVTREE_DTS)/$(notdir $(TARGET_DTS))
 
+$(DEVTREE_DTC): | $(SRC_ROOT)
```

Comment:
that dependency doesn't look right, I suggest splitting this target in 2 (similar way to the other targets in the same file):
```Makefile
DTC_SRC = ....

$(DTC_SRC): | $(SRC_ROOT)
	$(call EXTRACT_FILE,dtc-$(DTC_TAG).tar.gz,$(MD5_SUM_dtc-$(DTC_TAG)))
	chmod -R a-w $(DTC_SRC)

$(DEVTREE_DTC): $(DTC_SRC)
	$(MAKE) -C $(SRC_ROOT)/dtc-$(DTC_TAG) NO_PYTHON=1
```

Reply:
done